### DISCUSSION TITLE
Sidebar collapsed by default

### DISCUSSION THREAD

**Discussion by heather-timmins** [vikramlearning/blazorbootstrap#699] [Q&A] [answered]
Hi, 

Sorry if i've missed something but can you make the sidebar collapsed by default? and only toggle to open? 

Thanks

![image](https://github.com/vikramlearning/blazorbootstrap/assets/144691290/5a9d0420-e25e-4979-8364-2f67c78792bc)

**Comment by gvreddy04** [comment]
@heather-timmins Thank you for using BlazorBootstrap. I will address enhancement in issue https://github.com/vikramlearning/blazorbootstrap/issues/700

**Reply by heather-timmins** [reply, marked as answer]
@gvreddy04  Hey, I have found a way round this! I just call ToggleSidebar(); in the SidebarDataProvider and it works
